src/main/java/dev/jlibra/example/GetAccountStateExample.java

```diff
@@ -23,7 +23,7 @@
 
 public class GetAccountStateExample {
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) throws IOException, InterruptedException {
```

Comment:
This one does not have the Thread.sleep added, should it have? Or why was he exception added?

Reply:
Yes, removed it.